### DISCUSSION TITLE
Problem with fetching data from pocketbase rest api

### DISCUSSION THREAD

**Discussion by Nymuxyzo** [pocketbase/pocketbase#4775] [Q&A] [answered]
This is my collection
![pocketbase_table](https://github.com/pocketbase/pocketbase/assets/1729839/9d386dd3-4563-428b-ad9d-327b9c88c06b)
The api rules
![pocketbase_api_rules](https://github.com/pocketbase/pocketbase/assets/1729839/394cbcad-91e5-4b34-b378-4e3b52b20612)
The result from the http request:
![pocketbase_query_result2](https://github.com/pocketbase/pocketbase/assets/1729839/a28e8a6e-f297-442b-ba81-4fbd2268e200)

HTTP request:
```
GET http://127.0.0.1:8090/api/collections/employees/records HTTP/1.1
Content-Type: application/json
Authorization: TOKEN
```

Why am I not receiving the email for the second person? I don't know where the error lies. I have completely reset pocketbase, but that didn't help either.

Version 0.22.9

**Comment by ganigeorgiev** [comment, marked as answer]
It is working correctly. The `email` filed will be returned only if at least one of the conditions is met:

- the user has `emailVisibility` set to `true`
- the authenticated user is the owner
- the authenticated user is admin

**Reply by Nymuxyzo** [reply]
@ganigeorgiev thanks for your quick reply. I have one more question. With or without the API Update Rule, users can't edit other people's information, can they? (I don't mean admins from pocketbase's point of view.) This is based on the same constellation as my first question. Thank you very much!
